app/views/players/show.html.slim

```diff
@@ -19,47 +18,47 @@ table.player
         | WINS
       td.bar.bg-blue[rowspan="3"]
       td.black.align-left.pad-left.score
-        = player.score.wins
+        = @player.score.wins
       td.dark-grey.align-right.stat
         | GOALS
       td.bar.bg-red[rowspan="3"]
       td.black.align-left.pad-left.score
-        = player.score.goals
+        = @player.score.goals
     tr
       td.dark-grey.align-right.stat
         | LOSES
       td.black.align-left.pad-left.score
-        = player.score.loses
+        = @player.score.loses
       td.dark-grey.align-right.stat
         | WIN RATIO
       td.black.align-left.pad-left.score
-        = player.score.wRatio.round(2)
+        = @player.score.wRatio.round(2)
     tr
       td.dark-grey.align-right.stat
         | POINTS
       td.black.align-left.pad-left.score
-        = player.score.points
+        = @player.score.points
       td.dark-grey.align-right.stat
         | PULL-UPS
       td.black.align-left.pad-left.score
-        = player.score.pullUps
+        = @player.score.pullUps
```

Comment:
just a note for the future, the naming convention for ruby is underscore `pull_ups`